assets/src/components/nav/navMenu.tsx

```diff
@@ -45,6 +46,10 @@ const NavMenu: React.FC<Props> = ({ mobileMenuIsOpen, toggleMobileMenu }) => {
             <OldCloseIcon className="c-nav-menu__close-icon" />
           </button>
         </div>
+        <div className="c-nav-menu__logged-in-as">
```

Comment:
I'm not attached to this by any stretch - I wanted to try this out and see how hard it was given the menu items right now, but it's possible that it'll need to be redone when https://app.asana.com/0/1148853526253420/1206526459943486/f is merged.